### PR TITLE
Fix for: Password: De-duplicate ldap drivers code

### PR DESCRIPTION
**Pull request by army1349**

### CODE REVIEW

File: plugins/password/drivers/ldap_simple.php [hunk 1]
```diff
@@ -228,8 +228,8 @@ function search_userdn($rcmail, $ds)
 
         $this->_debug("S: OK");
 
-        $search_base   = rcube_ldap_password::substitute_vars($search_base);
-        $search_filter = rcube_ldap_password::substitute_vars($search_filter);
+        $search_base   = substitute_vars($search_base);
+        $search_filter = substitute_vars($search_filter);
```
Comment (alecpl) [inline plugins/password/drivers/ldap_simple.php]: That should be self::substitute_vars().

File: plugins/password/drivers/ldap_simple.php [hunk 2]
```diff
@@ -31,7 +31,7 @@ class rcube_ldap_simple_password
 {
     private $debug = false;
     private $user;
-    private $conn;
+    protected $conn;
```
Comment (alecpl) [inline plugins/password/drivers/ldap_simple.php]: Let's make all these protected.

File: plugins/password/drivers/ldap_simple.php (hunk 1 above)
Comment (army1349) [reply plugins/password/drivers/ldap_simple.php]: My bad.

File: plugins/password/drivers/ldap_simple.php (hunk 2 above)
Comment (army1349) [reply plugins/password/drivers/ldap_simple.php]: OK

**Comment by army1349** [comment]
Thanks!

**Pull request merged**